SPECIFICATION.md

```diff
@@ -25,8 +25,8 @@ For the purposes of brevity, this document refers to the following terms which a
 This document will use the following terms to define data types.
 1. **NUMERIC**: The **NUMERIC** data type is a sequence of integers between 0 and 99999999, inclusive.
 2. **STRING**: The **STRING** data type is a sequence of UTF-8, [NFC Normalized](https://www.unicode.org/faq/normalization.html) characters, up to 255 bytes after encoding.
-3. **HASH**: The **HASH** data type is a sequence of 64 alphanumeric characters containing a hexadecimal hash.
-3. **SIGNATUREHEX**: The **SIGNATUREHEX** data type is a sequence of up-to-72 alphanumeric characters containing a hexadecimal digest.
+3. **HASH**: The **HASH** data type is a sequence of 52 alphanumeric characters containing a base32-encoded hash.
+3. **SIGNATUREBASE32**: The **SIGNATUREBASE32** data type is a sequence of up-to-72 alphanumeric characters containing a base32url-encoded digest.
```

Comment:
The signature can be up to 73 bytes, right? And the base32 for that will be what, about 40% larger?

Reply:
Updated to 102 bytes. 512 bits / 5 bits per char in Base32